reference/content-types/category_list.rst

```diff
@@ -0,0 +1,25 @@
+Category list
+=============
+
+Description
+-----------
+
+Shows a list of all available categories. The user can select with a checkbox
+which ones to assign to the page. Categories can be managed in the settings
+section of Sulu. The selection will be saved as an array.
+
+Parameters
+----------
+
+No parameters available
+
+Example
+-------
+
+.. code-block:: xml
```

Comment:
Definitely, but that would be another PR for me. This documentation is already better than nothing :-)